app/js/controllers/main.js

```diff
@@ -23,6 +25,20 @@ export default class MainController extends BaseController {
       'reminders': remindersController,
     };
 
+    const speechController = new SpeechController();
+    speechController.on(
```

Comment:
I would move this block earlier in the constructor so that the `speechController` instance can be passed as an option to the controllers (just like `mountNode`). Then it'll be up to the controllers to do whatever they want with it.